### PR TITLE
fix: no rating submitted text fixed

### LINKED ISSUE #14934: UI: Text spacing on insights ratings section is whacky
Screenshot 2024-05-07 at 9 10 30 PM

### PR DESCRIPTION
**Pull request by smitgol**
## What does this PR do?

- Fixes #14934
- Fixes CAL-3660

Before 
image

After:
no_rating_fixes

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected)
- [x] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com)
- [x] I have added or modified automated tests that prove my fix is effective or that my feature works (PRs might be rejected if logical changes are not properly tested)

### CODE REVIEW

File: packages/ui/components/empty-screen/EmptyScreen.tsx [hunk 1]
```diff
@@ -65,11 +65,7 @@ export function EmptyScreen({
         )}
         {!customIcon ? null : <>{customIcon}</>}
         <div className={`flex ${limitWidth ? "max-w-[420px]" : ""}  flex-col items-center`}>
-          <h2
-            className={classNames(
-              "text-semibold font-cal text-emphasis text-center text-xl",
-              icon && "mt-6"
-            )}>
+          <h2 className={classNames("text-emphasis text-center text-xl font-bold", icon && "mt-6")}>
```
Comment (Udit-takkar) [inline packages/ui/components/empty-screen/EmptyScreen.tsx:68]: why did you remove the font?

Comment (smitgol) [reply packages/ui/components/empty-screen/EmptyScreen.tsx:68]: font-cal was causing spacing between letters.I tried to reduce letter but was not working so i removed that font

Comment (PeerRich) [reply packages/ui/components/empty-screen/EmptyScreen.tsx:68]: the issue is not font-cal the issue is that tremor has a "numeric" class above

Comment (PeerRich) [inline packages/ui/components/empty-screen/EmptyScreen.tsx:68]:
```suggestion
          <h2 className={classNames("text-semibold font-cal text-emphasis text-center text-xl normal-nums", icon && "mt-6")}>
```

File: packages/ui/components/empty-screen/EmptyScreen.tsx [hunk 2]
```diff
@@ -65,11 +65,7 @@ export function EmptyScreen({
         )}
         {!customIcon ? null : <>{customIcon}</>}
         <div className={`flex ${limitWidth ? "max-w-[420px]" : ""}  flex-col items-center`}>
-          <h2
-            className={classNames(
-              "text-semibold font-cal text-emphasis text-center text-xl",
-              icon && "mt-6"
-            )}>
+          <h2 className={classNames("text-semibold font-cal text-emphasis text-center text-xl normal-nums", icon && "mt-6")}>
```
Comment (PeerRich) [inline packages/ui/components/empty-screen/EmptyScreen.tsx:68]:
```suggestion
          <h2
            className={classNames(
              "text-semibold font-cal text-emphasis text-center text-xl normal-nums",
              icon && "mt-6"
            )}>
```

**Review by PeerRich — APPROVED** [review]

**Pull request merged**